Cargo.toml

```diff
@@ -14,11 +14,11 @@ edition = "2018"
 
 [features]
 default = ["native"]
-native = ["runtime-native"]
+native = ["runtime-attributes/native", "runtime-native"]
 
 [dependencies]
 futures-preview = "0.3.0-alpha.16"
-runtime-attributes = { path = "runtime-attributes", version = "0.3.0-alpha.5" }
+runtime-attributes = { path = "runtime-attributes", version = "0.3.0-alpha.5", default-features = false }
```

Comment:
Question? Doesn't this disable `runtime-native` for all installations by default, unless opted into?

I just ran tests, and it seems to still work. But I'm not sure if I quite understand what's going on, and want to make sure runtime continues to work out of the box. Thanks!

Reply:
https://github.com/rustasync/runtime/blob/a78d1a4c675b253dbd17016c461748282c01de2b/Cargo.toml#L15-L17

Default feature set of `runtime` contains `native`, so `runtime-native` and `runtime-attributes/native` will be enabled by default AFAIK.